packages/tx-ingestion/src/factories.ts

```diff
@@ -0,0 +1,18 @@
+import { JsonRpcProvider, Provider } from '@ethersproject/providers'
```

Comment:
This file needs to be deduplicated as well, I think it should live in the test-utils in one of the typescript packages

Reply:
This is pretty similar to: https://github.com/ethereum-optimism/optimism-monorepo/blob/d4eb073b7c485ec3ab62db53bd2067abf47b80f7/packages/contracts/src/deployment/config.ts#L24